<?xml version="1.0" ?>
<launch>

  <arg name="manager"	default=""/>

  <include file="$(find aist_aruco_ros)/launch/multicam.launch">
    <arg name="manager"	value="$(arg manager)"/>
  </include>

  <arg if="$(eval manager == '')"	name="command"	value="standalone"/>
  <arg unless="$(eval manager == '')"	name="command"	value="load"/>
  <node name="camera_calibrator" pkg="nodelet" type="nodelet"
  	args="$(arg command) aist_camera_calibration/camera_calibrator
	      $(arg manager)">
    <remap from="/point_correspondences_set"
	   to="/aruco_multi_detector/point_correspondences_set"/>
  </node>

</launch>
